scripts/apps/authoring/widgets/views/authoring-widgets.html

```diff
@@ -1,44 +1,11 @@
-<div class="pinned-tab__wrapper">
-    <section class="tabcontent">
-        <!--
-            react based widgets use WidgetHeading component
-            that has pinning button already inside
-
-            this was done to prevent absolutely positioned pinning button
-            from covering other components
-        -->
-        <div
-            ng-if="pinnedWidget.component == null"
-            class="tabcontent-actions"
-        >
-            <button class="icn-btn active" ng-click="pinWidget(widget)">
-                <i class="icon-pin"></i>
-            </button>
-        </div>
-
-        <div
-            ng-if="pinnedWidget.component == null"
-            sd-widget
-            data-widget="pinnedWidget"
-            data-configurable="pinnedWidget.configurable"
-            data-save="pinnedWidget.save()"
-        >
-            <div ng-include="pinnedWidget.template" class="sd-wrap-helper"></div>
-        </div>
-
-        <sd-widget-react
-            ng-if="pinnedWidget.component != null"
-            widget="pinnedWidget"
-            article="item"
-            class="sd-widget"
-        ></sd-widget-react>
-    </section>
-</div>
-
-<div class="widget-wrapper"
+<div
+    class="widget-wrapper"
     ng-class="{
         'active-right': active,
-        'right-extended': active.extended}">
+        'right-extended': active.extended
+    }"
+>
+    <!-- RIGHT SIDEBAR START -->
```

Comment:
Widget itself is also a part of the sidebar, so the name isn't that good. I'd suggest "available widgets list start/end"